### ISSUE TITLE
Export missing. Version 0.22

### ISSUE THREAD

**Issue by RobinHerzog** [hwchase17/langchainjs#216]
Hello,

I get the following error:

 ```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './text_splitter' is not defined by "exports" in ...node_modules/langchain/package.json
```

for 
```
import { RecursiveCharacterTextSplitter } from "langchain/text_splitter";
```

**Comment by nfcampos** [comment]
Hi, it looks like maybe you are missing `"type": "module"` in your package.json, see more detailed install instructions here https://hwchase17.github.io/langchainjs/docs/getting-started

**Comment by danieldunderfelt** [comment]
I am also having this issue in an ESM node project.

**Comment by nfcampos** [comment]
Are you using Typescript? If so there's some additional setup instructions here https://hwchase17.github.io/langchainjs/docs/install

**Comment by danieldunderfelt** [comment]
That didn't work. I am working on a project created with create-t3-app. I tried setting the target to ES2020 (it was es2017) but no luck. I am trying to run a script that uses Langchain with the "tsx" package.

**Comment by Elvincth** [comment]
> That didn't work. I am working on a project created with create-t3-app. I tried setting the target to ES2020 (it was es2017) but no luck. I am trying to run a script that uses Langchain with the "tsx" package.

Same here, I followed the doc, and still no luck.

**Comment by nfcampos** [comment]
Hi, we've released a new version 0.0.49 which adds a CJS build and fixes this issue.
